app/javascript/packages/badge-tooltip/badge-tooltip-element.ts

```diff
@@ -0,0 +1,65 @@
+import { tooltip } from '@18f/identity-design-system';
+
+class BadgeTooltipElement extends HTMLElement {
+  badge: HTMLElement;
+
+  connectedCallback() {
+    this.badge = this.querySelector('.usa-tooltip')!;
+
+    this.setUpTooltip();
+    this.badge.addEventListener('mouseover', () => this.handleHover());
```

Comment:
How does someone access the tooltip content if they're using a keyboard or screen reader?

Reply:
I'm not sure, should those be tab-focusable? It seemed like from Kamal's comment they would only be a hover
https://www.figma.com/design/RbqHzdFNiCMTXSKSBOqwYX?node-id=4482-9726#769800201
It leaves Mobile out of the picture also.

Reply:
The way I would expect it to work is that the badge is focusable (which it is currently), and that the tooltip is visible as long as the badge has focus. In addition to the current hover behavior.